HISTORY.rst

```diff
@@ -3,6 +3,17 @@
 Release History
 ---------------
 
+2.1.1 (2023-08-02) Avondale
```

Comment:
I think we don't need to name a bugfix release.

Reply:
Also, you forgot to bump the version in __init__.py.

Reply:
Dang... I need to make myself a checklist.